### DISCUSSION TITLE
Sample IQ file and Waterfall Display

### DISCUSSION THREAD

**Discussion by DrAnonymous** [bemasher/rtlamr#271] [General]
I have an SDRplay RSP1a and have tried multiple times to decode meter data. I have gas, water, and electrical meters that should all be transmitting. The FCC ids are: EWQ100WD / EWQ100GDLAS / R7PEG1R1S6.

Can I send someone an IQ file to see if they find anything? Here's a waterfall plot of one of the ISM frequencies and I think the short horizontal lines are different meters transmitting. Are those meters like I thought?

![ISMband2](https://user-images.githubusercontent.com/10542209/216780600-4a682ba8-8d42-4be7-a79d-111749bcf12b.jpg)

I am running on windows 11, using sdrplay-rtltcp.

**Comment by bemasher** [comment]
I have a test fixture for decoding IQ files, send it along. Can you also
tell me the sample rate and bit depth?

**Comment by DrAnonymous** [comment]
[SDRuno_20230205_021452Z_913600kHz.zip](https://github.com/bemasher/rtlamr/files/10610079/SDRuno_20230205_021452Z_913600kHz.zip)

I believe its 12bit at 2Ms/s.

**Comment by bemasher** [comment]
There don't appear to be any rtlamr-compatible messages in that sample. It's possible the recording wasn't long enough to capture anything, but the messages present look FSK modulated.

**Comment by DrAnonymous** [comment]
I grabbed a larger sample, but its too large to post here. I sent an email with a link to drive. Sampled around 917Mhz for about 10 seconds.

**Reply by bemasher** [reply]
Sorry, no obvious messages in that one either.

**Reply by DrAnonymous** [reply]
Drat! Thank you for looking, I appreciate it.

**Comment by DrAnonymous** [comment]
Would someone mind posting a short IQ file with an IDM message in it? I hope to learn what it should look like in a waterfall display as well as work on understanding decoding in general.
